From 74b6017139e9b80169fa52bf9b0523d928d90119 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Fri, 30 Mar 2012 23:42:55 +0300
Subject: [PATCH 108/311] fix description of on_replied()

on_replied() checks the value of c2_rpc_item::ri_error, not
c2_conf_fetch_resp::fr_rc.
---
 conf/confc.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/conf/confc.c b/conf/confc.c
index b0d770f..8208fea 100644
--- a/conf/confc.c
+++ b/conf/confc.c
@@ -313,12 +313,12 @@
  * c2_rpc_post().  (Not now; see @ref confc-offline.)
  *
  * A state machine remains in S_WAIT_REPLY state until a reply from
- * confd arrives. This event triggers on_replied() callback.  If the
- * error code contained in confd's reply (c2_conf_fetch_resp) is
- * non-zero, on_replied() posts an AST that will eventually move the
- * state machine to S_FAILURE state.  If the error code is zero,
- * on_replied() uses c2_rpc_item_get() to "claim" the rpc item and
- * schedules transition to S_GROW_CACHE state.
+ * confd arrives. This event triggers on_replied() callback.  If
+ * c2_rpc_item::ri_error is non-zero, on_replied() posts an AST that
+ * will eventually move the state machine to S_FAILURE state.  If
+ * ->ri_error is zero, on_replied() increments rpc item's reference
+ * counter (c2_rpc_item_get()) and posts an AST, scheduling transition
+ * to S_GROW_CACHE state.
  *
  * - S_WAIT_STATUS
  *
-- 
1.8.3.2

